__tests__/app.test.js

```diff
@@ -111,3 +111,42 @@ describe('GET /api/articles/:article_id', () => {
         });
     });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with all comments for an article', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.comments.length).toBe(2)
```

Comment:
Nice and important assertion here as you are using a for each to make assertions in this test too 👍